libraries/AP_NavEKF2/AP_NavEKF2_PosVelFusion.cpp

```diff
@@ -359,19 +378,21 @@ void NavEKF2_core::FuseVelPosNED()
             velHealth = ((velTestRatio < 1.0f)  || badIMUdata);
             // declare a timeout if we have not fused velocity data for too long or not aiding
             velTimeout = (((imuSampleTime_ms - lastVelPassTime_ms) > gpsRetryTime) || PV_AidingMode == AID_NONE);
-            // if data is healthy  or in constant velocity mode we fuse it
-            if (velHealth || velTimeout) {
+            // use velocity data if healthy, timed out, or in constant position mode
+            if (velHealth || velTimeout || (PV_AidingMode == AID_NONE)) {
```

Comment:
Aiding mode was already checked in velTimeout above, this new one will never be used.